Source/Core/Core/DolphinAnalytics.h

```diff
@@ -21,86 +21,86 @@
 enum class GameQuirk
 {
   // Several Wii DI commands that are rarely/never used and not implemented by Dolphin
-  USES_DVD_LOW_STOP_LASER,
-  USES_DVD_LOW_OFFSET,
-  USES_DVD_LOW_READ_DISK_BCA,  // NSMBW known to use this
-  USES_DVD_LOW_REQUEST_DISC_STATUS,
-  USES_DVD_LOW_REQUEST_RETRY_NUMBER,
-  USES_DVD_LOW_SER_MEAS_CONTROL,
+  UsesDVDLowStopLaser,
+  UsesDVDLowOffset,
+  UsesDVDLowReadDiskBCA,  // NSMBW known to use this
+  UsesDVDLowRequestDiscStatus,
+  UsesDVDLowRequestRetryNumber,
+  UsesDVDLowSerMeasControl,
 
   // Dolphin only implements the simple DVDLowOpenPartition, not any of the variants where some
   // already-read data is provided
-  USES_DIFFERENT_PARTITION_COMMAND,
+  UsesDifferentPartitionCommand,
 
   // IOS has implementations for ioctls 0x85 and 0x89 and a stub for 0x87, but
   // DVDLowMaskCoverInterrupt/DVDLowUnmaskCoverInterrupt/DVDLowUnmaskStatusInterrupts
   // are all stubbed on the PPC side so they presumably will never be used.
   // (DVDLowClearCoverInterrupt is used, though)
-  USES_DI_INTERRUPT_MASK_COMMAND,
+  UsesDIInterruptMaskCommand,
 
   // Some games configure a mismatched number of texture coordinates or colors between the transform
   // and TEV/BP stages of the rendering pipeline. Currently, Dolphin just skips over these objects
   // as the hardware renderers are not equipped to handle the case where the registers between
   // stages are mismatched.
-  MISMATCHED_GPU_TEXGENS_BETWEEN_XF_AND_BP,
-  MISMATCHED_GPU_COLORS_BETWEEN_XF_AND_BP,
+  MismatchedGPUTexGensBetweenXFAndBP,
+  MismatchedGPUColorsBetweenXFAndBP,
 
   // The WD module can be configured to operate in six different modes.
   // In practice, only mode 1 (DS communications) and mode 3 (AOSS access point scanning)
   // are used by games and the system menu respectively.
-  USES_UNCOMMON_WD_MODE,
+  UsesUncommonWDMode,
 
-  USES_WD_UNIMPLEMENTED_IOCTL,
+  UsesWDUnimplementedIOCtl,
```

Comment:
`IOCtl` is capitalized in various ways throughout the codebase; this one makes sense to me but 🤷.